guide/server.mdx

```diff
@@ -0,0 +1,207 @@
+---
+sidebar_position: 2
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Server Integration
+
+## Install the SDK
+
+Install the SDK for the language of your choice. We provide libraries for Node, Python and Elixir(?).  
```

Comment:
**issue (blocking):** Let's not forget about removing this or the question mark itself.
```suggestion
Install the SDK for the language of your choice. We provide libraries for Node, and Python.  
```